extensions/arc/src/models/controllerModel.ts

```diff
@@ -50,6 +50,13 @@ export class ControllerModel {
 		this._onInfoUpdated.fire(this._info);
 	}
 
+	public get azdataAdditionalEnvVars() {
+		return {
+			'KUBECONFIG': this.info.kubeConfigFilePath,
+			'KUBECTL_CONTEXT': this.info.kubeClusterContext
```

Comment:
it is called cluster context not kubectl context.

Reply:
Are you saying the variable name is: CLUSTER_CONTEXT?

Reply:
double-checked with ADZDATA sources, the environment variable name is KUBECTL_CONTEXT